tests/rip7560/test_rip7560.py

```diff
@@ -38,6 +41,147 @@ def test_eth_sendTransaction7560_valid1(w3, wallet_contract, tx_7560):
     w3.eth.get_transaction(rethash)
 
 
+def test_system_event_success(
+    w3, entry_point_rip7560, factory_contract_7560, paymaster_contract_7560, tx_7560
+):
+    tx_7560.paymaster = paymaster_contract_7560.address
+    tx_7560.factory = factory_contract_7560.address
+    tx_7560.factoryData = factory_contract_7560.functions.createAccount(
+        ADDRESS_ZERO, 123, ""
+    ).build_transaction({"gas": 1000000})["data"]
+    new_sender_address = factory_contract_7560.functions.getCreate2Address(
+        ADDRESS_ZERO, 123, ""
+    ).call()
+    tx_7560.sender = new_sender_address
+    tx_7560.nonce = hex(0)
+
+    res = tx_7560.send()
+    send_bundle_now()
+    receipt = w3.eth.get_transaction_receipt(res.result)
+
+    system_event_args, _, _ = get_system_events(entry_point_rip7560)
+
+    system_event_topics = list(
+        filter(
+            lambda x: x.address == "0x0000000000000000000000000000000000007560",
+            receipt.logs,
+        )
+    )[0].topics
+
+    assert system_event_args == {
+        "sender": new_sender_address,
+        "paymaster": paymaster_contract_7560.address,
+        "deployer": factory_contract_7560.address,
+        # pylint: disable=fixme
+        # TODO: also pass and check 'nonceKey'
+        "nonce": 0,
+        "success": True,
+        # pylint: disable=fixme
+        # TODO: pass and check real 'actualGasCost'
+        "actualGasCost": 0,
```

Comment:
not checking exact gas you can use `pytest.approx()`

Reply:
Removed `actualGasCost` from the event.